assets/shaders/alphamask.frag.glsl

```diff
@@ -1,5 +1,3 @@
-#version 120
```

Comment:
yes it is, but it also has to be in the first not comment line and it is not allowed to be defined twice. What I did is adding a possibility to add common functions before the real shader code. To be able to do it, the version header can not be in the shader file.

instead it is added in the code. see https://github.com/spjoe/openage/commit/c12aff570d96f42a1cf5011df0147f9b05c2c41f#diff-76e3d56a04902eefda9a5140e3af8bbeR69